diesel/src/query_builder/upsert/on_conflict_actions.rs

```diff
@@ -69,6 +91,25 @@ where
     }
 }
 
+impl<DB, T>
+    QueryFragment<DB, crate::backend::sql_dialect::on_conflict_clause::MysqlLikeOnConflictClause>
```

Comment:
This impl should be moved to `mysql/query_builder/upsert.rs` or something like that, as it's mysql specific.

Reply:
I think I see what you're saying but, again, I'm working off [this example](https://github.com/diesel-rs/diesel/blob/42413462fbb67d6b4b612f1f62ba9f8af465d920/diesel/src/query_builder/returning_clause.rs#L30-L54) provided earlier which appears to have `Pg*` traits/structs in a general module

Sorry if I'm misunderstanding though